query/sexp/parser.go

```diff
@@ -190,7 +190,7 @@ func getIdentString(tree *peg.ExpressionTree) string {
 		}
 	} else {
 		if tree.Value != '"' {
-			out += string(tree.Value)
+			out += string([]rune{rune(tree.Value)})
```

Comment:
I think converting `->rune->string` directly (without a slice) would work as well.

Reply:
That throws an error when you try to build - saying that string(rune) does not have the intended behavior in most cases. Actually makes "go test" fail, so that was why I changed it here.